src/WebJobs.Extensions.NotificationHubs/Bindings/NotificationHubAttributeBindingProvider.cs

```diff
@@ -59,9 +59,12 @@ public Task<IBinding> TryCreateAsync(BindingProviderContext context)
 
             Func<string, INotificationHubClientService> invokeStringBinder = (invokeString) => service;
 
+            var converterManager = _jobHostConfig.GetService<IConverterManager>();
+            converterManager = converterManager.AddNotificationHubConverters(attribute.Platform);
```

Comment:
Will hold off on this change until we have support converter manager at attribute level